crates/stackable-versioned/tests/basic.rs

```diff
@@ -5,14 +5,16 @@ use stackable_versioned::Versioned;
 #[versioned(
     version(name = "v1alpha1"),
     version(name = "v1beta1"),
-    version(name = "v1")
+    version(name = "v1"),
```

Comment:
Can we please have a comment like this at the start of the file?
```
// To see generated code run `cd crates/stackable-versioned` and afterwards
// `cargo expand --test basic`
```

Reply:
Added in 6d5ab16